deploy/deploy.js

```diff
@@ -0,0 +1,144 @@
+const fs = require("fs");
+const AdmZip = require("adm-zip");
+const glob = require("glob");
+const process = require("process");
+const jwt = require("jsonwebtoken");
+
+const { get, post, put } = require("./utils/httpRequests");
+const { config, secrets } = require("./utils/config");
+const authenticate = require("./utils/authenticate");
+const deployArmTemplate = require("./utils/deployArmTemplate");
+const { azureResourceUrl } = require("./utils/resourceHelpers");
+const deployCosmos = require("./utils/deployCosmos");
+const asyncTimeout = require("./utils/asyncTimeout");
+
+(async function deploy() {
+    try {
+        console.log(`Beginning deployment to ${process.env["DeployEnvironment"]}`)
+
+        let armSessionToken = await authenticate("https://management.core.windows.net/");
+        let storageSessionToken = await authenticate("https://storage.azure.com/");
+
+        // Infrastructure
```

Comment:
Why is all of this code commented out? It seems that as of now, it only is deploying the infrastructure. None of the other templates in `definitions` folder apart from `main.template.json` are being used. If they are not required, we can remove them.